ui/components/app/snaps/snap-content-footer/snap-content-footer.js

```diff
@@ -41,13 +40,13 @@ export default function SnapContentFooter({ snapName, snapId }) {
         color={IconColor.iconMuted}
         paddingRight={1}
       />
-      <Typography color={TextColor.textMuted} variant={TypographyVariant.H7}>
+      <Text color={TextColor.textMuted} variant={TextVariant.bodyXs} as="h6">
         {t('snapContent', [
           <Button type="inline" onClick={handleNameClick} key="button">
```

Comment:
Need to update the component API here and import the constants from the component-library

```suggestion
          <Button variant={BUTTON_VARIANT.LINK} size={BUTTON_SIZES.INHERITY} onClick={handleNameClick} key="button">
```

Reply:
Will definitely do that @georgewrmarshall 
Thanks a lot for patiently reviewing my PRs.